# HG changeset patch
# User Sergey Mironov <grrwlf@gmail.com>
# Date 1381140533 -14400
#      Mon Oct 07 14:08:53 2013 +0400
# Node ID bebfcb78ac223ad43b967187a52cce534ffeef53
# Parent  04109cf0642a38d234a4efc7d31c76bc4bf22c02
Add <dl> <dt> <dd> tags

diff --git a/lib/ur/basis.urs b/lib/ur/basis.urs
--- a/lib/ur/basis.urs
+++ b/lib/ur/basis.urs
@@ -979,6 +979,19 @@
   -> tag ([Colspan = int, Rowspan = int] ++ tableAttrs)
          ([Tr] ++ other) ([Body] ++ other) [] []
 
+(** Definition lists *)
+
+val dl : other ::: {Unit} -> [other ~ [Body,Dl]]
+  => unit
+  -> tag [] ([Body] ++ other) ([Dl] ++ other) [] []
+
+val dt : other ::: {Unit} -> [other ~ [Body,Dl]]
+  => unit
+  -> tag [] ([Dl] ++ other) ([Body] ++ other) [] []
+
+val dd : other ::: {Unit} -> [other ~ [Body,Dl]]
+  => unit
+  -> tag [] ([Dl] ++ other) ([Body] ++ other) [] []
 
 (** Aborting *)
 
